python/lsst/jointcal/jointcal.py

```diff
@@ -189,8 +222,9 @@ def run(self, dataRefs):
             prof = pstats.Stats('jointcal_load_catalog.prof')
             prof.strip_dirs().sort_stats('cumtime').print_stats(20)
         else:
+            old_wcss = []
```

Comment:
The name `old_wcss` is naughty.  First, it's breaking naming rules (camelCase has been adopted in this file, so shouldn't be using under_scores).  Second, it's breaking English rules (should be -es instead of -s).
I suggest `oldWcsList`.